[php7] Unable to build from source

May affect other packages that depend on curl as well.
```
checking for cURL support... yes
checking for libcurl.pc... using default path
checking for cURL 7.15.5 or greater... 7.66.0
checking for SSL support in libcurl... yes
checking how to run the C preprocessor... cc -E
checking for openssl support in libcurl... no
checking for gnutls support in libcurl... no
checking for curl_easy_perform in -lcurl... no
configure: error: There is something wrong. Please check config.log for more information.
php7 failed to build: `./configure --prefix=/usr/local --docdir=/usr/local/doc --infodir=/usr/local/info --libdir=/usr/local/lib --localstatedir=/usr/local/tmp --mandir=/usr/local/share/man --sbindir=/usr/local/bin --with-config-file-path=/usr/local/etc --with-libdir=lib --with-freetype-dir=/usr/local/include/freetype2/freetype --with-pcre-regex=/usr/local/lib --with-jpeg-dir=/usr/local/lib --with-kerberos=/usr/local/lib --with-png-dir=/usr/local/lib --with-webp-dir=/usr/local/lib --with-xpm-dir=/usr/local/lib --enable-exif --enable-fpm --enable-ftp --enable-mbstring --enable-opcache --enable-pcntl --enable-shared --enable-shmop --enable-sockets --enable-zip --with-bz2 --with-curl --with-gd --with-gettext --with-gmp --with-libzip --with-mysqli --with-openssl --with-pcre-regex --with-pdo-mysql --with-pear --with-readline --with-tidy --with-unixODBC --with-xsl --with-zlib` exited with 1
```

--- a/packages/curl.rb
+++ b/packages/curl.rb
@@ -3,21 +3,21 @@ require 'package'
 class Curl < Package
   description 'Command line tool and library for transferring data with URLs.'
   homepage 'https://curl.haxx.se/'
-  version '7.66.0'
+  version '7.66.0-1'
   source_url 'https://curl.haxx.se/download/curl-7.66.0.tar.xz'
   source_sha256 'dbb48088193016d079b97c5c3efde8efa56ada2ebf336e8a97d04eb8e2ed98c1'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/curl-7.66.0-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '135488fd75e5127645d4ffe07d67e635229e5bfdb238852e5d8af9c10fd5f15d',
-     armv7l: '135488fd75e5127645d4ffe07d67e635229e5bfdb238852e5d8af9c10fd5f15d',
-       i686: 'c03cb9fe9b2d30e106a96a686ce79e367281cea8296241c6151938fcecde27e0',
-     x86_64: 'aa67936bd49b3722c356c5fd5137f5b3b350f6085a8da19d70e03ee58f428e60',
+    aarch64: 'd3fa586deef7030c2bdd3c3e477391c276b34276d5dddbf755cf4785a56ec482',
+     armv7l: 'd3fa586deef7030c2bdd3c3e477391c276b34276d5dddbf755cf4785a56ec482',
+       i686: '76dfc54131f9ad33fdc6fba5ffc74eabd730322ba88cfdf10ded79b304c1ce22',
+     x86_64: 'bee317b81814d931317a7539cd86678e28b3de46508aa09ad7a5aa648ba761ff',
   })
 
   depends_on 'groff' => :build

